Ports/SDL2/patches/0001-Add-SerenityOS-platform-support.patch

```diff
@@ -1662,3 +1690,6 @@ index 0000000000000000000000000000000000000000..f74e1bd981734caf023b77a9b9fa30c4
 +#endif /* SDL_serenityvideo_h_ */
 +
 +/* vi: set ts=4 sw=4 expandtab: */
+-- 
+2.40.1
```

Comment:
Please tun the patch through `./package.sh dev` once (just enter and exit should be enough) to get rid of the short form object IDs and the Git version trailer.